app/helpers/application_helper.rb

```diff
@@ -3,4 +3,14 @@ module ApplicationHelper
   def error_message_on(object, attribute)
     raw content_tag :span, object.errors.full_messages_for(attribute).to_sentence, class: 'alert-form'
   end
+
+  def link_to_add_images(name, f, association)
```

Comment:
@jynti Why is this code in the application helper? Please use better argument names and refactor this method.